src/test/java/pl/jakubkozlowski/learning/firststeps/descriptor/UserTestDescriptor.java

```diff
@@ -0,0 +1,14 @@
+package pl.jakubkozlowski.learning.firststeps.descriptor;
+
+public class UserTestDescriptor {
```

Comment:
it should not be called descriptor. IT is not descriptor, just simple constants for testing purposes